packages/plexus/src/zoom/MiniMap.tsx

```diff
@@ -14,15 +14,17 @@
 
 import * as React from 'react';
 
-import resetZoomIcon from './resetZoomIcon';
-
+import { FaMagnifyingGlassPlus, FaMagnifyingGlassMinus } from 'react-icons/fa6';
+import { HiMiniArrowsPointingOut } from 'react-icons/hi2';
```

Comment:
it's preferable to use icons from the same package - I see HiMagnifyingGlassMinus in hi2

Reply:
they are also MIT license, unlike FA

Reply:
Ok will make that change